placement-rules-in-sql.md

```diff
@@ -0,0 +1,154 @@
+>
+> ```sql
+> mysql> show placement labels;
+> +--------+----------------+
+> | Key    | Values         |
+> +--------+----------------+
+> | disk   | ["ssd"]        |
+> | region | ["us-east-1"]  |
+> | zone   | ["us-east-1a"] |
+> +--------+----------------+
+> 3 rows in set (0.00 sec)
+> ```
+
+| Option Name                | Description                                                                                    |
+|----------------------------|------------------------------------------------------------------------------------------------|
+| `PRIMARY_REGION`           | Raft leaders will be placed in stores which have the `region` label that match this value.     |
+| `REGIONS`                  | Raft followers will be placed in stores which have the `region` label that matches this value. |
+| `SCHEDULE`                 | The strategy used to schedule the placement of followers. Either `EVEN` (default) or `MAJORITY_IN_PRIMARY`. |
+| `FOLLOWERS`                | The number of followers. For example, `FOLLOWERS=2` means that there will be 3 copies of the data (2 followers and 1 leader). |
+| **Advanced configuration**                                                                                                  |
+| `CONSTRAINTS`              | A list of constraints that apply to all roles. For example, `CONSTRAINTS="[+disk=ssd]`.        |
+| `FOLLOWER_CONSTRAINTS`     | A list of constraints that only apply to followers.                                            |
+
+## Examples
+
+### Increasing the number of replicas
+
+The default configuration of [`max-replicas`](/pd-configuration-file.md#max-replicas) is `3`. To increase this for a specific set of tables, you can use a placement policy as follows:
+
+```sql
+CREATE PLACEMENT POLICY fivereplicas FOLLOWERS=4;
+CREATE TABLE t1 (a INT) PLACEMENT POLICY=fivereplicas;
+```
+
+Note that the PD configuration includes the leader and follower count, thus 4 followers + 1 leader equals five replicas in total.
+
+To expand on this example, the placement for the followers can also be described using the `PRIMARY_REGION` and `REGIONS` placement options:
+
+```sql
+CREATE PLACEMENT POLICY eastandwest PRIMARY_REGION="us-east-1" REGIONS="us-east-1,us-east-2,us-west-1" SCHEDULE="MAJORITY_IN_PRIMARY" FOLLOWERS=4;
+CREATE TABLE t1 (a INT) PLACEMENT POLICY=eastandwest;
+```
+
+The `SCHEDULE` instructs TiDB how to balance the followers. The default schedule of `EVEN` ensures a balance of followers in all regions. The schedule `MAJORITY_IN_PRIMARY` can be used to ensure that enough followers are placed in the primary region (`us-east-1`) that quorum can be achieved. This helps provide lower latency transactions at the expense of availability should the primary region completely fail.
```

Comment:
Does the `region` here mean data center, instead of the "Region" that implies a data range in TiKV?
```suggestion
The `SCHEDULE` option instructs TiDB on how to balance the followers.

The default schedule of `EVEN` ensures a balance of followers in all regions. You can use the `MAJORITY_IN_PRIMARY` schedule to ensure that enough followers are placed in the primary region (`us-east-1`) so that quorum can be achieved. If the primary region completely fails, you can the `MAJORITY_IN_PRIMARY` schedule for lower latency transactions at the expense of availability.
```